qutebrowser/commands/runners.py

```diff
@@ -172,6 +172,20 @@ def parse(self, text, *, aliases=True, fallback=False, keep=False):
                 return self.parse(new_cmd, aliases=False, fallback=fallback,
                                   keep=keep)
         try:
+            
+            """ If the command given has only one completion match, replace
+                the given command with the match.
+                Ex: If they type "bac" and the only completion is "back", 
+                turn the command into "back".
+            """
```

Comment:
Please use a normal comment - docstrings are intended as documentations at the start of a function/class